spec.json

```diff
@@ -1054,19 +1064,6 @@
                             ]
                         }
                     ]
-                },
-                {
-                    "propertyName": "Color",
-                    "values": [
-                        { "name" : "NoColor",  "value" : "0", "type" : "int", "description": [ "No color." ] },
-                        { "name" : "Black",    "value" : "1", "type" : "int", "description": [ "Black color." ] },
-                        { "name" : "Blue",     "value" : "2", "type" : "int", "description": [ "Blue color." ] },
-                        { "name" : "Green",    "value" : "3", "type" : "int", "description": [ "Green color." ] },
-                        { "name" : "Yellow",   "value" : "4", "type" : "int", "description": [ "Yellow color." ] },
-                        { "name" : "Red",      "value" : "5", "type" : "int", "description": [ "Red color." ] },
-                        { "name" : "White",    "value" : "6", "type" : "int", "description": [ "White color." ] },
-                        { "name" : "Brown",    "value" : "7", "type" : "int", "description": [ "Brown color." ] }
-                    ]
```

Comment:
My apologies for that, beginner's error.  Do I need to do anything to undo it?  And then I guess I need to figure out the right one.  Is that a case of locating it, re-uploading and submitting a new PR?